CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Changelog
 
+### AMI Release v20230919
+* amazon-eks-gpu-node-1.28-v20230919
+* amazon-eks-gpu-node-1.27-v20230919
+* amazon-eks-gpu-node-1.26-v20230919
+* amazon-eks-gpu-node-1.25-v20230919
+* amazon-eks-gpu-node-1.24-v20230919
+* amazon-eks-gpu-node-1.23-v20230919
+* amazon-eks-arm64-node-1.28-v20230919
+* amazon-eks-arm64-node-1.27-v20230919
+* amazon-eks-arm64-node-1.26-v20230919
+* amazon-eks-arm64-node-1.25-v20230919
+* amazon-eks-arm64-node-1.24-v20230919
+* amazon-eks-arm64-node-1.23-v20230919
+* amazon-eks-node-1.28-v20230919
+* amazon-eks-node-1.27-v20230919
+* amazon-eks-node-1.26-v20230919
+* amazon-eks-node-1.25-v20230919
+* amazon-eks-node-1.24-v20230919
+* amazon-eks-node-1.23-v20230919
+
+[Release versions](https://docs.aws.amazon.com/eks/latest/userguide/eks-linux-ami-versions.html) for these AMIs:
+* `1.28.1-20230919`
+* `1.27.5-20230919`
+* `1.26.8-20230919`
+* `1.25.13-20230919`
+* `1.24.17-20230919`
+* `1.23.17-20230919`
+
+Binaries used to build these AMIs are published:
+* s3://amazon-eks/1.28.1/20230914/
+* s3://amazon-eks/1.27.5/20230914/
+* s3://amazon-eks/1.26.8/20230914/
+* s3://amazon-eks/1.25.13/20230914/
+* s3://amazon-eks/1.24.17/20230914/
+* s3://amazon-eks/1.23.17/20230914/
+
+AMI details:
+* `kernel`:
+  * Kubernetes 1.23 and below: 5.4.254-170.358.amzn2
+  * Kubernetes 1.24 and above: 5.10.192-183.736.amzn2
+  * **Note** that the GPU AMI on Kubernetes 1.27 and below will continue to use kernel-5.4 as we work to address a [compatibility issue](https://github.com/awslabs/amazon-eks-ami/issues/1222) with `nvidia-driver-latest-dkms`.
```

Comment:
I don't think we have a resolution in sight for this, so we should probably just say:
```suggestion
  * **Note** that the GPU AMI on Kubernetes 1.27 and below will continue to use kernel-5.4 due to a [compatibility issue](https://github.com/awslabs/amazon-eks-ami/issues/1222) with `nvidia-driver-latest-dkms`.
```